src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -447,6 +450,11 @@ else if (workBuilding != building)
             }
             else if (job != null)
             {
+                EntityCitizen citizen = getCitizenEntity();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'citizen' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)